pxblue/symbols
# Feature/blui 5601 symbols GitHub ci

Fixes BLUI-5601# .



#### Changes proposed in this Pull Request:

- add blui-ci github workflow
- remove circleCI config and stop building
- update readme with new badge
- remove circleCI references from publish readme
- add tagging job



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- if you want to see jobs run make small commit and revert it or see jobs here
- https://github.com/etn-ccis/blui-symbols/actions/runs/8896179342



#### Any specific feedback you are looking for?

-
